README.md

```diff
@@ -1,5 +1,5 @@
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
-[![Build Status](https://travis-ci.com/usi-verification-and-security/opensmt.svg?branch=master)](https://travis-ci.com/usi-verification-and-security/opensmt)
+[![Build Status](https://circleci.com/gh/usi-verification-and-security/opensmt.svg?style=svg)](https://circleci.com/gh/usi-verification-and-security/opensmt?branch=master)
```

Comment:
The URL did not lead me to the master branch. The following worked for me:
```suggestion
[![Build Status](https://circleci.com/gh/usi-verification-and-security/opensmt.svg?style=svg)](https://app.circleci.com/pipelines/github/usi-verification-and-security/opensmt?branch=master)
```

Reply:
Fixed